e2e/nomostest/new.go

```diff
@@ -181,11 +181,9 @@ func SharedTestEnv(t nomostesting.NTB, opts *ntopts.New) *NT {
 		// Reset cached metrics for each test
 		ReconcilerMetrics: make(testmetrics.ConfigSyncMetrics),
 		// Reset git-repo port-forward for each test
-		gitRepoPort:    0,
-		gitRepoPodName: "",
```

Comment:
Why remove these?

Yes, the empty string is default, but I wanted to make it clear these are being reset on each test.

Reply:
The pod name is scoped to a function closure now, so we don't need to keep track of each of these in the higher level NT struct. It's still reset on each test